src/cbc_sdk/platform/alerts.py

```diff
@@ -563,9 +570,9 @@ def _update_threat_workflow_status(self, state, remediation, comment):
             request["remediation_state"] = remediation
         if comment:
             request["comment"] = comment
-        url = "/api/alerts/v7/orgs/{0}/alerts/workflow".format(self._cb.credentials.org_key)
+        url = "/appservices/v6/orgs/{0}/threat/workflow/_criteria".format(self._cb.credentials.org_key)
```

Comment:
add a line to set threat id
        request["threat_id"] = [self.threat_id]

This is what the UI was generating, plus a v7 update for existing alerts
{
    "threat_id": [
        "CFED0B211ED09F8EC1C83D4F3FBF1709"
    ],
    "state": "OPEN",
    "comment": "",
    "remediation_state": null
}